roles/base_os/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+base_os_use_firewalld: "{{ False if ('openshift_master' in role_names or 'openshift_node' in role_names) else True }}"
```

Comment:
We prefer to keep all roles as generic as possible. This makes the base_os role 'know' about the openshift_master and openshift_node roles. Also, if we want to add more exclusions in the future, we'd have to add more to this list, which gets ugly quickly.

Instead, consider just passing in this variable to this role in the specific config playbooks for the openshift_master and openshift_node host types.